1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md

```diff
@@ -0,0 +1,198 @@
+# Terminal Wio
```

Comment:
The hardware is called the Wio Terminal, so should the translation use a translated version of the name, or stick to the product name?

Reply:
We should stick to the product name. I thought the "brand name" was Wio and it was a terminal, this is why I translated that way. 
I will update the translation to be Wio Terminal. ;)

Reply:
@leocamello - let me know when you have updated the device name and I'll get this merged.

Reply:
Done, @jimbobbennett ;)

Reply:
Awesome, thanks!